components/Java/h2o_predictor/src/test/main/java/com/parallelm/components/h2o_predictor/H2OPredictorTest.java

```diff
@@ -174,6 +174,40 @@ public void testH2OPredict() throws Exception{
             Assert.fail("Fail");
         }
     }
-}
 
+    @Test
+    public void testH2OPredictResults() throws Exception{
+        String output_file = "/tmp/out-mini.csv";
+
+        try {
+            out.println("Test 7: testH2OPredictResults in progress.. ");
```

Comment:
To tell the truth: why do you want to print "Test NN" is running.. bla bla?
- Test framework should print it by itself.
- And so why to print "Test passed"? If there are no errors, means it passed. And if not - it failed.
- You hardcoded func names into several messages. You could use 
smth like this instead 
https://stackoverflow.com/questions/473401/get-name-of-currently-executing-test-in-junit-4
maybe something like this also available for the junit you use...

Reply:
@yakov-g that's an excellent suggestion, let me work on making these changes